scripts/publish_to_bintray.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import requests
+from requests.auth import HTTPBasicAuth
+import os, sys
+import json
+
+bintray_auth = (os.environ['BINTRAY_USER'], os.environ['BINTRAY_PASS'])
+
+
+def create_version(version):
```

Comment:
https://github.com/kiwix/kiwix-lib/blob/aa7f61c354ae6ab28829696e84e1cd7f29cf520a/android-kiwix-lib-publisher/kiwixLibAndroid/publish.gradle I am not seeing the depth of information that we used to be supplying here.
We - I think - have to be supplying the groupId at least and the artefact name.
I think the generation of a pom file is necessary? To be honest this is the first artefact I've uploaded so I don't have huge knowledge here

Reply:
The new version of the script also publish the pom file.